src/fsharp/TastOps.fs

```diff
@@ -4632,14 +4632,23 @@ and remapValReprInfo g tmenv (ValReprInfo(tpNames, arginfosl, retInfo)) =
 
 and remapValData g tmenv (d: ValData) =
     let ty = d.val_type
-    let topValInfo = d.val_repr_info
+    let topValInfo = d.ValReprInfo
     let ty' = ty |> remapPossibleForallTy g tmenv
+    let val_declaring_entity = d.DeclaringEntity |> remapParentRef tmenv
```

Comment:
change these names, e.g. `let declaringEntity' = ` or some other systematic naming convention to indicate old/new values in the remap.  I don't particularly like `'` so you can use `let declaringEntityR` or something where `R` means `remapped` etc

Reply:
yes was going to suggest the same the tick makes it often disappear in debugger